packages/datadog-instrumentations/src/http/server.js

```diff
@@ -10,6 +10,7 @@ const shimmer = require('../../../datadog-shimmer')
 const startServerCh = channel('apm:http:server:request:start')
 const errorServerCh = channel('apm:http:server:request:error')
 const finishServerCh = channel('apm:http:server:request:finish')
+const closeServerCh = channel('apm:http:server:request:close')
```

Comment:
Sometimes "close" event is launched and "finish" is not. For example, if the browser cancels the request (e.g. multiple fast refresh in the same page)
We need to clear our overhead controller quotas at this point because the finish event may not be launched.

Reply:
Maybe we don't need the finish event then. Let's just replace it with the close one ?

Reply:
This is one thing that I want to investigate, but it looks like a "big" change that can broke something. So I didn't want to remove `finish` event in that PR.

Reply:
There is also the opposite case of `finish` happening without `close` if the connection is kept alive. We should probably try to combine them somehow.

Reply:
I have added a test which failed in the previous approach, and currently is working. I have removed `finish` event and now we are only listening to `close`

`integration-tests/closed-connection/closed-connection.spec.js`